psi4/src/psi4/libmints/molecule.cc

```diff
@@ -1940,14 +1942,14 @@ Matrix *Molecule::inertia_tensor() const {
 
     for (i = 0; i < natom(); i++) {
         // I(alpha, alpha)
-        temp(0, 0) += mass(i) * (y(i) * y(i) + z(i) * z(i));
-        temp(1, 1) += mass(i) * (x(i) * x(i) + z(i) * z(i));
-        temp(2, 2) += mass(i) * (x(i) * x(i) + y(i) * y(i));
+        temp(0, 0) += mass(i, true) * (y(i) * y(i) + z(i) * z(i));
```

Comment:
Ghost atoms shouldn't effect moments of inertia, so assuming all of these should return zero mass.

Reply:
Including the mass of ghosted atoms here would lead to incorrect rotational constants, but there may be some use case (ie when orienting if you want A-Gh(B) you want it to be consistent with A-B/Gh(A-)B counterparts).

Same thing would go for the COM calculation should there be optionals for how ghosts are treated in these functions as well?

Reply:
I was a little worried about these sorts of use cases. Presets could harm n-body / fragmentation schemes in general (or manual counterpoise corrections), or anyone trying to do "apples to apples" comparisons on systems with potentially different COM's, no matter which way you do them. An optional would be my first thought Py-side, not sure about here. Any other opinions?

Reply:
This is changing current behavior, yes? Sounds dangerous. Except for when you're correcting existing ghost misunderstandings, I'd encode existing behavior rather than higher notions of what it should be. The optional arg you could add to inertia_tensor then allows that higher definition to be got, while the existing code stays the same.

Reply:
I'll go ahead and add an optional for `inertia_tensor` and `center_of_mass` defaulting to `false` (current behavior), then I'll probably do some testing to see whether current tests even care what the default is (my guess is these functions are probably not tested with ghost atoms). Either way, I like this solution.